### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by harjaapdhillon16**

### CODE REVIEW

**Comment by Megha-Dev-19** [comment]
@harjaapdhillon16  Could you please provide description about the changes?
thanks

File: src/components/pages/home/OgSbtApplication.js [hunk 1]
```diff
@@ -1,10 +1,230 @@
...
+                  method: "sbt_mint",
+                  args: {
+                    receiver: person.wallet_identifier,
+                    metadata: {
+                      ttl: "",
+                      memo: "",
+                    },
+                  },
+                });
+                log_event({
```
Comment (Megha-Dev-19) [inline src/components/pages/home/OgSbtApplication.js:130]: is log event browser side or server side?

File: src/components/pages/home/applyCommunityVerify/verifyPhoneAndEmail.js [hunk 2]
```diff
@@ -4,6 +4,8 @@ import PhoneInput, { isPossiblePhoneNumber } from "react-phone-number-input";
 import { toast } from "react-toastify";
 import OtpInput from "react-otp-input";
 import { checkUniquePhone } from "../../../../utils/uniqueUser";
+import { log_event } from "../../../../utils/utilityFunctions";
+import { api_link } from "../../../../utils/supabase";
```
Comment (Megha-Dev-19) [inline src/components/pages/home/applyCommunityVerify/verifyPhoneAndEmail.js:8]: why do we have it in underscore naming, is there a predefined naming conventions that we are following?  can we follow camelCase

**Review by Megha-Dev-19 — COMMENTED** [review]
I see that naming of files is also different, what's the convention which we are following for it?

File: src/components/pages/home/applyCommunityVerify/verifyPhoneAndEmail.js (hunk 2 above)
Comment (harjaapdhillon16) [reply src/components/pages/home/applyCommunityVerify/verifyPhoneAndEmail.js:8]: The underscore naming conventions is followed for non react code and every other hook and component follows camelCase this keeps the code visually separated !

File: src/components/pages/home/OgSbtApplication.js (hunk 1 above)
Comment (harjaapdhillon16) [reply src/components/pages/home/OgSbtApplication.js:130]: The log Event stores events in our database for every action that the user takes that leads to the changes in DB

**Review by Megha-Dev-19 — APPROVED** [review]

**Pull request merged**